qbraid_qir/qasm3/visitor.py

```diff
@@ -14,81 +14,31 @@
 """
 import copy
 import logging
-import sys
 from abc import ABCMeta, abstractmethod
 
 # pylint: disable=too-many-instance-attributes,too-many-lines
 from collections import deque
-from typing import Optional, Union
+from typing import Union
 
 import pyqir
 import pyqir._native
 import pyqir.rt
-from openqasm3.ast import (
-    AliasStatement,
-    ArrayLiteral,
-    ArrayType,
-    BinaryExpression,
-    BooleanLiteral,
-    BoolType,
-    BranchingStatement,
-    ClassicalArgument,
-    ClassicalAssignment,
-    ClassicalDeclaration,
-    ConstantDeclaration,
-    DiscreteSet,
-    DurationLiteral,
-    ExpressionStatement,
-    FloatLiteral,
-)
-from openqasm3.ast import FloatType as Qasm3FloatType
-from openqasm3.ast import (
-    ForInLoop,
-    FunctionCall,
-    GateModifierName,
-    Identifier,
-    ImaginaryLiteral,
-    Include,
-    IndexedIdentifier,
-    IndexExpression,
-    IntegerLiteral,
-)
-from openqasm3.ast import IntType as Qasm3IntType
-from openqasm3.ast import (
-    IODeclaration,
-    QuantumBarrier,
-    QuantumGate,
-    QuantumGateDefinition,
-    QuantumGateModifier,
-    QuantumMeasurementStatement,
-    QuantumReset,
-    QubitDeclaration,
-    RangeDefinition,
-    ReturnStatement,
-    Span,
-    Statement,
-    SubroutineDefinition,
-    SwitchStatement,
-    UnaryExpression,
-    WhileLoop,
-)
 from pyqir import BasicBlock, Builder, Constant
 from pyqir import IntType as qirIntType
 from pyqir import PointerType
 
 from .elements import Context, InversionOp, Qasm3Module, Variable
 from .exceptions import Qasm3ConversionError
-from .oq3_maps import (
+from .utils.expressions import Qasm3ExprEvaluator
+from .utils.imports import *
```

Comment:
Instead of doing a wildcard import of a module filled with explicit imports, let's just use a namespace import. See comment above.